.azure-pipelines/windows/intellinav.yml

```diff
@@ -0,0 +1,10 @@
+steps:
+- task: vsintellinav.vsck-service-endpoint.build-task.upload-cache-build-task@0
+  displayName: 'VS IntelliNav Upload'
+  inputs:
+    vsckEndpoint: 'azureappservice-dev' # name of the IntelliNav service connection
```

Comment:
azureappservice?

Reply:
It was just the generic name I used for the service connection.  I didn't have the foresight to think I'd be doing this on all the extensions 😇

Reply:
Can we change it?

Reply:
Yes, I can.  What would you like to name it?

Reply:
idk something with either azuretools or azcode instead of appservice

Reply:
`azazurecodetools` it is